ci/test-stable.sh

```diff
@@ -46,6 +46,16 @@ for program in programs/native/*; do
   )
 done
 
+# Run program/native/bpf_loader's test and bench with bpf_c feature
+(
+  set -x
+  cd "programs/native/bpf_loader"
+  echo --- program/native/bpf_loader test --features=bpf_c
+  cargo test --verbose --features="bpf_c"
+  echo --- program/native/bpf_loader bench --features=bpf_c
```

Comment:
This could be part of the previous block where all the native programs are run but it would require all the native programs to add a `bpf_c` feature to their Cargo.toml files.  Opinions?